chapel.html.markdown

```diff
@@ -201,8 +201,6 @@ select( inputOption ){
   otherwise { 
     writeln( "Any other Input" );
     writeln( "the otherwise case doesn't need a do if the body is one line" );
-    writeln( "Oh, and when statements dont cascade like the case statements" );
```

Comment:
I felt it was redundant given the comment above
"However, Select statements don't cascade like in C or Java"

It may not be necessary to remove output.